pbs/pbsrequest.go

```diff
@@ -67,23 +67,22 @@ func (bidder *PBSBidder) LookupBidID(Code string) string {
 }
 
 type PBSRequest struct {
-	AccountID     string            `json:"account_id"`
-	Tid           string            `json:"tid"`
-	TimeoutMillis uint64            `json:"timeout_millis"`
-	AdUnits       []AdUnit          `json:"ad_units"`
-	UserIDs       map[string]string `json:"user_ids"`
-	IsDebug       bool              `json:"is_debug"`
+	AccountID     string          `json:"account_id"`
+	Tid           string          `json:"tid"`
+	CacheMarkup   int8            `json:"cache_markup"`
+	Secure        int8            `json:"secure"`
```

Comment:
There's JSON wonkiness around bool vs int8 - openRTB uses int8 across the board so I'm sticking with that for consistency.